content/test/unit/repository/DeploymentsRepository.spec.ts

```diff
@@ -0,0 +1,456 @@
+import { DeploymentsRepository } from '@katalyst/content/repository/extensions/DeploymentsRepository'
+import { Entity } from '@katalyst/content/service/Entity'
+import { EntityType, EntityVersion, SortingField, SortingOrder } from 'dcl-catalyst-commons'
+import { anything, capture, deepEqual, instance, mock, spy, verify, when } from 'ts-mockito'
+import MockedDataBase from './MockedDataBase'
+
+describe('DeploymentRepository', () => {
+  let repository: DeploymentsRepository
+  let db: MockedDataBase
+
+  beforeEach(() => {
+    db = mock(MockedDataBase)
+    repository = new DeploymentsRepository(instance(db) as any)
+  })
+
+  describe('areEntitiesDeployed', () => {
+    beforeEach(() => {
+      db = mock(MockedDataBase)
+      repository = new DeploymentsRepository(instance(db) as any)
+    })
+
+    describe('when the entities list is not empty', () => {
+      const dbResult = ['1', '2']
+
+      beforeEach(() => {
+        when(db.map(anything(), anything(), anything())).thenReturn(Promise.resolve(dbResult))
+      })
+
+      it('should return a map of entity id to the deployment status', async () => {
+        const entities = ['1', '3']
+        const result = await repository.areEntitiesDeployed(entities)
+
+        expect(result).toEqual(
```

Comment:
I'd remove this as the only thing that you are testing is the mock

Reply:
This is asserting on the logic of the map creation in the return statement of the function

Reply:
My bad!